gdx/src/com/badlogic/gdx/math/FastNoiseLite.java

```diff
@@ -0,0 +1,2353 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.math;
+
+// MIT License
```

Comment:
I'm pretty sure we're totally fine on licensing here as long as the MIT license and its authorship credit are intact. I also believe libGDX already uses MIT-licensed code somewhere, because so much open source code uses it.